src/test/java/org/jboss/forge/arquillian/container/provider/ContainerDirectoryParserTest.java

```diff
@@ -0,0 +1,82 @@
+package org.jboss.forge.arquillian.container.provider;
+
+import org.assertj.core.api.SoftAssertions;
+import org.jboss.forge.arquillian.container.model.Container;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+
+public class ContainerDirectoryParserTest {
+
+    private List<Container> containers;
+
+    @Before
+    public void loadContainers() throws IOException {
+        ContainerDirectoryParser containerDirectoryParser = new ContainerDirectoryParser();
+        containerDirectoryParser.containerDirectoryLocationProvider = new FileContainerIndexLocationProvider();
+        containers = containerDirectoryParser.getContainers();
```

Comment:
Why not just calling get in each and every test instead of having `containers` as a field? Describes exactly what is component under test and what is the interaction.